src/platyPS/platyPS.psm1

```diff
@@ -104,6 +104,8 @@ function New-MarkdownHelp
 
     begin
     {
+        validateWorkingProvider
+
```

Comment:
we are using C#-like code-style, which suggests use blank lines only after closing `}` in the method bodies.
Please, remove blank line.